imio/dms/mail/tests/test_table.py

```
@@ -80,6 +87,34 @@ def test_AssignedGroupColumn(self):
         col = AssignedGroupColumn(self.portal, self.portal.REQUEST, None)
         self.assertEqual(col.renderCell(task).encode("utf8"), "Direction générale")
 
+    def test_SignRequestVersionsTable(self):
+        activate_signing(self.portal)
+        request, files = create_sign_request(self.portal, oid="sr-versions")
```

Ruff (0.16.1)

[warning] Unpacked variable `files` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)